lip9/caip2.md

```diff
@@ -0,0 +1,59 @@
+---
+namespace-identifier: lip9-caip2
+title: LIP9 Namespace, aka Lisp - Chains
+author: Simon Warta (@webmaster128)
+discussions-to: https://github.com/ChainAgnostic/CAIPs/issues/7, https://github.com/ChainAgnostic/CAIPs/pull/1
+status: Draft
+type: Standard
+created: 2019-12-05
+updated: 2020-01-16
+requires: CAIP-2
+supersedes: CAIP-6
+---
+
+# CAIP-2
+
+*For context, see the [CAIP-2][] specification.*
```

Comment:
Link missing

Reply:
That's a fancy form of markdown-reference that I learned from celehner!

Reply:
at the bottom, [CAIP-2]: https://... is defined, and this lets you update all occurrences of that link in a document centrally.  I'm over here re-centralizing these specs!